magefiles/shellx/async-cmd.go

```diff
@@ -0,0 +1,35 @@
+package shellx
```

Comment:
It's weird that we need to implement that by ourselves - aren't there any helpers e.g. from Mage which facilitate that? 🤔

Reply:
not that I liked 😄 I wanted to have:
- an option to write `hugo server -p 60123` instead of `"hugo", "server", "-p", "60123"`
- there was no option to "Start" the command, only `Run` (in general makes sens, but here it would require usage of goroutines which will complicate the code)